client/src/redux/sagas/jobPostingSaga.js

```diff
@@ -49,9 +49,11 @@ function* submitJobPosting({ payload }) {
   }
 }
 
```

Comment:
Tässä Sagassa pitäis tuossa submitJobPostingissa käyttää tuota pois-kommentoitua jobPostingia